src/App.tsx

```diff
@@ -1,34 +1,24 @@
-import {useState} from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import "./App.css";
+import { Counter } from "./components/Counter.tsx";
+import { SearchForm } from "./components/SearchForm.tsx";
+import { GenreSelect } from "./components/GenreSelect.tsx";
 
 function App() {
-    const [count, setCount] = useState(0)
-    return (
-        <>
-            <div>
-                <a href="https://vite.dev" target="_blank" rel="noreferrer">
-                    <img src={viteLogo} className="logo" alt="Vite logo"/>
-                </a>
-                <a href="https://react.dev" target="_blank" rel="noreferrer">
-                    <img src={reactLogo} className="logo react" alt="React logo"/>
-                </a>
-            </div>
-            <h1>Vite + React</h1>
-            <div className="card">
-                <button onClick={() => setCount((count) => count + 1)}>
-                    count is {count}
-                </button>
-                <p>
-                    Edit <code>src/App.tsx</code> and save to test HMR
-                </p>
-            </div>
-            <p className="read-the-docs">
-                Click on the Vite and React logos to learn more
-            </p>
-        </>
-    )
+  const genres = ["ALL", "DOCUMENTARY", "COMEDY", "HORROR", "CRIME"];
+
+  return (
+    <div className="flex flex-col gap-4">
+      <SearchForm onSearch={console.log} initialQuery="Star Wars" />
+      <GenreSelect
+        genres={genres}
+        onSelect={console.log}
```

Comment:
You could actually change the selected genre instead of just console logging, but no problem - you can do it later.